code/modules/organs/subtypes/machine.dm

```diff
@@ -77,6 +77,9 @@
 		emp_counter--
 
 /obj/item/organ/cell/emp_act(severity)
+	if(owner.get_species() == "Military Frame")
```

Comment:
After discussing this in discord. Arrow, Paradox and others agree that HK should use surge module(an organ). But it should regenerate over time